packages/controllers/src/services/AbstractExecutionService.ts

```diff
@@ -96,6 +96,12 @@ export abstract class AbstractExecutionService<WorkerType>
       (snapId: string) => this.getRpcRequestHandler(snapId),
```

Comment:
We should stop exposing `getRpcRequestHandler`, by removing the action and marking the function as `private`.

Reply:
Removed actions now and also made this method private.